### DISCUSSION TITLE
Pagination logic inquiry

### DISCUSSION THREAD

**Discussion by arodriguezcortes** [twilio-labs/paste#1948] [Q&A] [answered]
Hi team. I just want to confirm. If I want to create a paged table with the new paste components I use the Table and Pagination components, but I should create the pagination logic myself, is that right? There's no implemented pagination logic inside Paste.

**Comment by kate-porter** [comment, marked as answer]
Hey @arodriguezcortes - yep, that's correct, the pagination logic is totally in your hands. Paste is purely presentational, so application logic and the way data is passed to the components is up to you.

**Reply by arodriguezcortes** [reply]
Thanks @katieporter !
